docs/advanced/navigating-politeia-data.md

```diff
@@ -73,12 +79,19 @@ The `comments.journal` file contains data related to comments. The table below d
 | **resultvotes**             |   |  
 
 
-### Identifying users
+### User data
+
+In the datasets presented here, **users are identified by their public keys**. On [proposals.decred.org](https://proposals.decred.org/), users are identified by their username (chosen when they created their public/private key pair). 
+
+Currently, to associate a public key with a username, you need to go through [proposals.decred.org](https://proposals.decred.org/). To look up the public key for a user, click on their username anywhere on the site. This will take you to the user's profile, which has a URL like this: 
+<https://proposals.decred.org/user/350a4b6c-5cdd-4d87-822a-4900dc3a930c>
+
+
+The final part of this URL is the Universally Unique ID (UUID) for the account. This can be input into a [public API](https://proposals.decred.org/api/v1/user/) exposed by the Politeia website, which will takes the UUID as an input and outputs user profile data, including the public key. For example, if you paste the above example UUID into a browser:
+<https://proposals.decred.org/api/v1/user/350a4b6c-5cdd-4d87-822a-4900dc3a930c>
```

Comment:
The newline after this URL needs to be removed, otherwise it makes the following comma look kinda weird:

![new](https://user-images.githubusercontent.com/6762864/48618257-ba012600-e990-11e8-938e-a25918af1eed.png)

Reply:
Done

Reply:
Ah crap I actually put the comment under the wrong url. It should have been the next one, so actually this issue still exists.

Reply:
Removed another newline. Is this what we're looking for?

![image](https://user-images.githubusercontent.com/1634777/48975782-6e541980-f02d-11e8-93fd-938f87bae536.png)